src/types/cwe.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+This file is part of CycloneDX JavaScript Library.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) OWASP Foundation. All Rights Reserved.
+*/
+
+import { isPositiveInteger, PositiveInteger } from './integer'
+
+/**
+ * Define the format for acceptable Common Weakness Enumeration (CWE) IDs.
+ * Refer to {@link https://cwe.mitre.org/index.html} for official specification.
+ * @see isCWE
+ */
+export type CWE = number | PositiveInteger
+
+export function isCWE (value: any): value is CWE {
```

Comment:
:question: is this correct according to the schema? is a CWE just a pos integer? 
Need to check XML and JSON schema

Reply:
it is. 
```json
"cwe": {
      "type": "integer",
      "minimum": 1,
      "title": "CWE",
      "description": "Integer representation of a Common Weaknesses Enumerations (CWE). For example 399 (of https://cwe.mitre.org/data/definitions/399.html)"
    },
```